src/traffic_server/RpcAdminPubHandlers.h

```diff
@@ -20,6 +20,8 @@
 
 #pragma once
 
+#include <iostream>
```

Comment:
Should this be included somewhere else? It doesn't look like the header is used in this file.